dotnetcore2.1-image/cookiecutter-aws-sam-hello-dotnet-lambda-image/{{cookiecutter.project_name}}/src/HelloWorld/HelloWorld.csproj

```diff
@@ -10,9 +10,9 @@
   </PropertyGroup>

```

Comment:
Why did integ tests fail?

Reply:
caused by docker request limit. The tests pass locally on my local machine, check the PR description for detail